sqrl-tools/sqrl-cli/src/main/java/com/datasqrl/compile/DockerCompose.java

```diff
@@ -8,8 +8,13 @@ public class DockerCompose {
 
   public static String getYml(Optional<Path> mountDir) {
     String volumneMnt = mountDir.map(dir -> dir.toAbsolutePath().normalize())
-        .map( dir -> "    volumes:\n"
-        + "      - " + dir.toAbsolutePath() + ":/build\n").orElse("");
+        .map(dir -> ""
+            + "    volumes:\n"
+            + "      - " + dir.toAbsolutePath() + ":/build\n"
+            + "      - ./init-flink.sh:/exec/init-flink.sh\n")
+        .orElse(""
+            + "    volumes:\n"
+            + "      - ./init-flink.sh:/exec/init-flink.sh\n");
```

Comment:
Can you make that a static string for the init-flink and then append the optional dir so you don't have to repeat the string?

Reply:
fixed so it's nicer